frame/treasury/src/benchmarking.rs

```diff
@@ -130,8 +130,8 @@ fn setup_bounty<T: Trait<I>, I: Instance>(u: u32, d: u32) -> (
 	Vec<u8>,
 ) {
 	let caller = account("caller", u, SEED);
-	let value: BalanceOf<T, I> = T::Currency::minimum_balance().saturating_mul(100.into());
-	let fee = T::Currency::minimum_balance().saturating_mul(2.into());
+	let value: BalanceOf<T, I> = T::BountyValueMinimum::get().saturating_mul(100.into());
```

Comment:
Why do we still multiply by 100 and now divide by 2 instead of multiplying by 2?

Reply:
Multiplying by 100 is arbitrary.

As for the fee, we were using the minimum balance there too, when we should be using the value of the bounty.

Of the value of the bounty, some portion is given to the end user, and some fee is given to the curator.

By having fee be value / 2, it means half goes to curator, half goes to the bounty person, which ensures that both people get a payout.